### PR TITLE
Search

### PR DESCRIPTION
**Pull request by rafaelbernard**
Proposal for showing a message when no results are found.

### CODE REVIEW

File: src/components/ModiteList/index.tsx [hunk 1]
```diff
@@ -104,7 +109,15 @@ const ModiteList: FunctionComponent<ModiteListProps> = ({
               lastScrollOffset={lastScrollOffsetModites}
             />
           )}
-          {!isDetails && !listRecords.length && <SkeletonList />}
+          {!isDetails && !listRecords.length && filter.length && (
+            <VirtualizedList
```
Comment (grgur) [inline src/components/ModiteList/index.tsx]: This is great, thanks for submitting the fix.

Just an idea: how about creating a custom component for `NoRecordsFound`?

e.g.

```jsx
const NoRecordsFound = () => {
  const moditeNotFound = {
    real_name: 'No modite found',
    tz: (Intl.DateTimeFormat().resolvedOptions().timeZone) // client timezone
  }
  
  return  <VirtualizedList {{...allTheOtherStuffHere}} />
}
```

Comment (rafaelbernard) [reply src/components/ModiteList/index.tsx]: Sure! Will do it.

**Comment by grgur** [comment]
@rafaelbernard any updates for this?

File: src/components/VirtualizedList/index.tsx [hunk 2]
```diff
@@ -29,11 +29,16 @@ const VirtualizedList: FunctionComponent<VirtualizedListProps> = ({
...
+      {(!addSpacerRow || (addSpacerRow && index !== 0)) && localRecords[index].id && (
         <Link to={`/details/${localRecords[index].id}`} className={`ListRow ${s.moditeRow}`} style={style}>
           <ModiteListItem item={localRecords[index]} key={localRecords[index].id} />
         </Link>
       )}
+      {(!addSpacerRow || (addSpacerRow && index !== 0)) && !localRecords[index].id && (
+        <span className={`ListRow ${s.moditeRow}`} style={style}>
+          <ModiteListItem item={localRecords[index]} />
+        </span>
+      )}
```
Comment (rafaelbernard) [inline src/components/VirtualizedList/index.tsx]: When does not have an id (e.g. no modite found) there is no detail link.

Comment (grgur) [reply src/components/VirtualizedList/index.tsx]: can you add that as a comment in the code?

Comment (rafaelbernard) [reply src/components/VirtualizedList/index.tsx]: Sure. Just dit!

Was very nice playing around React for the first time. Hope I will be able to contribute more.

**Pull request merged**